ruby/Gemfile.lock

```diff
@@ -0,0 +1,34 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    coderay (1.1.3)
+    dotenv (2.7.6)
+    ethon (0.13.0)
+      ffi (>= 1.15.0)
+    ffi (1.15.0)
+    hubspot-api-client (10.0.0)
+      json (~> 2.1, >= 2.1.0)
+      require_all (~> 3.0.0)
+      typhoeus (~> 1.4.0)
+    json (2.5.1)
+    method_source (1.0.0)
+    pry (0.14.0)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    require_all (3.0.0)
+    typhoeus (1.4.0)
+      ethon (>= 0.9.0)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  dotenv
+  hubspot-api-client (>= 10.0.0)
```

Comment:
Might be better to lock the client version on 10? (Since 11.0.0 might be with breaking changes)